src/foam/u2/view/TableCellFormatter.js

```diff
@@ -75,6 +75,11 @@ foam.CLASS({
     {
       class: 'Int',
       name: 'tableWidth'
+    },
+    {
+      class: 'String',
+      documentation: 'Column label that overrides the label property in table headers',
+      name: 'columnLabel'
```

Comment:
The expression: for this should just return the regular label. That would simplify line 60 above.